extensions/transform/org.eclipse.smarthome.transform.xslt/README.md

```diff
@@ -2,21 +2,33 @@
 
 Transform input using the XML Stylesheet Language for Transformations (XSLT).
 
-It expects the transformation rule to be read from a file which is stored under the `transform` folder. 
+XSLT is a standard method to transform an XML structures from one documentt into a new document with a other structure.
```

Comment:
A few grammar problems here.

Either "transform an XML structure" or "transform XML structures".
"document" (not "documentt")
"with another structure"